src/python/pants/core/util_rules/environments.py

```diff
@@ -180,6 +180,35 @@ class DockerEnvironmentTarget(Target):
     )
 
 
+class RemotePlatformField(StringField):
+    alias = "platform"
+    default = Platform.linux_x86_64.value
+    valid_choices = Platform
+    help = "The platform used by the remote execution environment."
+
+
+class RemoteEnvironmentTarget(Target):
+    alias = "_remote_environment"
+    core_fields = (
+        *COMMON_TARGET_FIELDS,
+        RemotePlatformField,
+    )
+    help = softwrap(
+        """
+        Configuration of a remote execution environment used for building your code, including the
+        environment variables and search paths used by Pants.
+
+        Note that you must also configure remote execution with the global options like
+        `remote_execution` and `remote_execution_address`.
+
+        Often, it is only necessary to have a single `_remote_environment` target for your
+        repository, but it can be useful to have >1 so that you can set different
+        `extra_platform_properties`. For example, with some servers, you could use this to
```

Comment:
And if you have different platform properties, you'll likely need different env vars, etc. So it's likely that more than one piece of the environment will be different if any are.